backend/typescript/models/volunteerSignUp.model.ts

```diff
@@ -0,0 +1,38 @@
+import mongoose, { Schema, Document } from "mongoose";
```

Comment:
We can get rid of this whole thing, this is using mongoose and we wanna stray away from mongoose and migrate everything to prisma

Reply:
Done!